pkgs/by-name/sp/spotube/package.nix

```diff
@@ -1,25 +1,28 @@
 {
-  lib,
-  stdenv,
-  fetchurl,
-
   autoPatchelfHook,
   dpkg,
+  fetchurl,
+  glib,
+  gtk3,
+  lib,
+  libGL,
+  libappindicator,
+  libnotify,
+  libsoup_3,
   makeBinaryWrapper,
   makeWrapper,
+  mesa,
+  mpv-unwrapped,
+  stdenv,
   undmg,
+  webkitgtk_4_1,
```

Comment:
Please restore the changes made to the ordering, as it's not necessary for this PR.

I generally only sort the `nativeBuildInputs` and `buildInputs` among themselves, not the whole list.

Reply:
The ordering should be restored, everything except `mpv-unwrapped` is removed from `mkLibraryPath`, and `LIBGL_ALWAYS_SOFTWARE` removed. I am on wayland and it still built and ran with this modifications.